docs/sources/operations/caching.md

```diff
@@ -67,16 +67,24 @@ To enable and configure Memcached:
              default_validity: 12h
        ```
     1. If the Loki configuration is used
-        1. Configure the chunk cache
+        1. Configure the chunk and index write cache
            ```yaml
            chunk_store_config:
              chunk_cache_config:
                memcached:
                  batch_size: 256
                  parallelism: 10
                memcached_client:
-                 host: <memcached host>
+                 host: <chunk cache memcached host>
                  service: <port name of memcached service>
+	   write_dedupe_cache_config: {
+             memcached:
+               batch_size: 100
+               parallelism: 100
+             memcached_client: 
+               host: <index writes memcached host>
+               service: <port name of memcached service>
+               consistent_hash: true
```

Comment:
Check the formatting on this example, in the preview the bit I've copy/pasted below isn't included in the code block.  

`write_dedupe_cache_config: { memcached: batch_size: 100 parallelism: 100 memcached_client: host: service: consistent_hash: true `